utils/launch_redis.py

```diff
@@ -37,15 +37,10 @@ def check_availability(n_nodes, port, udsport):
     is_uds = udsport is not None
     if is_uds:
         n_nodes = 1
-    cicd = os.getenv('SR_CICD_EXECUTION')
-    is_cicd = False if cicd is None else cicd.lower() == "true"
-    if is_cicd:
-        rediscli = 'redis-cli'
-    else:
-        rediscli = (
-            pathlib.Path(__file__).parent.parent
-            / "third-party/redis/src/redis-cli"
-        ).resolve()
+    rediscli = (
+        pathlib.Path(__file__).parent.parent
+        / "third-party/redis/src/redis-cli"
+    ).resolve()
```

Comment:
So again, this is removing a performance optimization that we intentionally left in. While I don't object, it does seem like it should be a separate PR